sentim_client.go

```diff
@@ -0,0 +1,141 @@
+package jamie_chicken
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+type emotionResponse struct {
+	EmotionScore emotionScore `json:"EmotionScore"`
+	Emotion      string       `json:"Emotion"`
+}
+
+type emotionScore struct {
+	Anger          float32 `json:"Anger"`
+	Neutral        float32 `json:"Neutral"`
+	Others         float32 `json:"Others"`
+	Happy          float32 `json:"Happy"`
+	Joy            float32 `json:"Joy"`
+	Affection      float32 `json:"Affection"`
+	Sad            float32 `json:"Sad"`
+	Disappointment float32 `json:"Disappointment"`
+}
+
+type conversation struct {
+	Conversation []string `json:"conversation"`
+	Lang         string   `json:"lang"`
+	IgnoreFirst  bool     `json:"ignore_first,omitempty"`
+}
+
+type sentimClient struct {
+	auth_token string
+	base_url   string
+	logger     *logrus.Logger
+}
+
+func NewSentimClient() *sentimClient {
+	l := logrus.New()
+	l.SetOutput(os.Stdout)
+	l.SetLevel(logrus.DebugLevel)
+	return &sentimClient{
+		base_url: "https://api.sentimllc.com/",
+		logger:   l,
+	}
+}
+
+func (c *sentimClient) IsGritty(message string) bool {
+	return c.isEmotion("Anger", message)
+}
+
+func (c *sentimClient) IsDisappointed(message string) bool {
+	return c.isEmotion("Disappointment", message)
+}
+
+func (c *sentimClient) isEmotion(emotion string, message string) bool {
+	emot, err := c.getEmotion(message)
+	if err != nil {
+		c.logger.Debugf("something went wrong with getting emotion from the api ¯\\_(ツ)_/¯")
+		c.logger.Errorf("I supposed you want to see the error: %s", err)
+		return false
+	}
```

Comment:
maybe we could set up a slack channel for logging...

Reply:
That is some responsible product ownership right there.

Reply:
https://github.com/fitzoh/jamie-chicken/pull/8
^^ thoughts?